README.adoc

```diff
@@ -472,6 +472,55 @@ spec:
 ...
 ----
 
+== Experimental
+
+=== Observing a Remote Cluster
+
+There are certain use cases where Kiali needs to be deployed in one cluster (Control Plane) and observe a different cluster (Data Plane). link:https://user-images.githubusercontent.com/6889074/87819080-ad099980-c839-11ea-834b-56eec038ce4d.png:[Diagram].
+
+Follow these steps:
+
+1: You should have one of the central istiod setups running
+
+  - https://github.com/istio/istio/wiki/Central-Istiod-manages-remote-data-plane
+
+  - https://github.com/istio/istio/wiki/Central-Istiod-single-cluster-steps
+
+2: Deploy the link:https://github.com/istio/istio/blob/master/samples/addons/kiali.yaml[Kiali ClusterRole, ClusterRoleBinding, and ServiceAccount] to the Data Plane
+
+3: Create a remote secret in the Control Plane, using the Data Plane ServiceAccount you just created. This allows the Control Plane to read from and modify the Data Plane
+[source,shell]
+----
+istioctl x create-remote-secret --service-account kiali-service-account --context=$DataPlane --name kiali | kubectl apply -n istio-system --context=$ControlPlane -f -
+----
+
+4: Run the Kiali Deployment with the following additions. Kiali will use the API server from the remote secret instead of the local API server.
```

Comment:
This step isn't clear.  also which cluster?

Reply:
Kiali would be deployed to the Control Plane cluster.

I expect someone to take the existing Kiali Deployment and then add the Volume and VolumeMounts specified. I'll say that explicitly